certbot/account.py

```diff
@@ -270,9 +270,12 @@ def _save(self, account, acme, regr_only):
                 if hasattr(acme.directory, "new-authz"):
                     regr = RegistrationResourceWithNewAuthzrURI(
                         new_authzr_uri=acme.directory.new_authz,
-                        body=regr.body,
-                        uri=regr.uri,
-                        terms_of_service=regr.terms_of_service)
+                        body="",
+                        uri=regr.uri)
+                else:
+                    regr = messages.RegistrationResource(
+                        body="",
```

Comment:
I used that because that's what we got back from the server when we queried new account with the only return existing argument, which seemed nicer to match.